Update tracing-dashboards.md (#1993)
SumoLogic/sumologic-documentation: docs/apm/traces/tracing-dashboards.md

The **APM: Service View** takes a contrary approach, displaying services by top level and breaking down their health by application. This view can be useful for shared services that support more than one application. The fourth level shows the top 50 most active operations performed on the selected service and application.

> The **APM: Environment View** displays environments at the top level and breaks down application health by environment (i.e., prod or dev, up to 10 values of **deployment.environment** tag). This view can be especially useful for understanding the top-down hierarchy of applications and services in a particular environment. The fourth level shows the top 50 most active operations executed on the environment, application, and service. <br/><img src={useBaseUrl('img/traces/Service-Dashboards-from-traces.png')} alt="Service Dashboards from traces" width="350"/>

Reviewer: ```suggestion
The **APM: Environment View** displays environments at the top level and breaks down application health by environment (such as `prod` or `dev`), with up to 10 values of **deployment.environment** tag. This view can be useful for understanding the top-down hierarchy of applications and services in a particular environment. The fourth level shows the top 50 most active operations executed on the environment, application, and service. <br/><img src={useBaseUrl('img/traces/Service-Dashboards-from-traces.png')} alt="Service Dashboards from traces" width="350"/>
```
